prow/plugins.yaml

```diff
@@ -240,8 +240,12 @@ config_updater:
 
 welcome:
 - repos:
-  - kubernetes/test-infra
-  message_template: "Welcome @{{.AuthorLogin}}! It looks like this is your first PR to {{.Org}}/{{.Repo}} 🎉🎉"
+   - kubernetes
```

Comment:
This looks like whitespace misalignment?

Reply:
Huh. This looks like a Suggested Changes bug. Will report internally.